src/visitor.ts

```diff
@@ -49,31 +73,57 @@ abstract class Visitor {
 
     }
 
-    // noinspection TsLint
     protected visitReturnStatement(node: ReturnStatement) {
+        const argument = node.argument;
 
+        if (argument !== null) {
+            this.visit(argument);
+        }
     }
 
-    // noinspection TsLint
     protected visitUnaryExpression(node: UnaryExpression) {
-
+        this.visit(node.argument);
     }
 
-    // noinspection TsLint
     protected visitBinaryExpression(node: BinaryExpression) {
-
+        this.visit(node.left);
+        this.visit(node.right);
     }
 
-    // noinspection TsLint
     protected visitIfStatement(node: IfStatement) {
+        this.visit(node.test);
+        this.visit(node.consequent);
 
+        if (node.alternate !== null) {
+            this.visit(node.alternate);
+        }
     }
 
-    // noinspection TsLint
     protected visitBlockStatement(node: BlockStatement) {
+        for (const statement of node.body) {
+            this.visit(statement);
+        }
+    }
 
+    protected visitVariableDeclaration(node: VariableDeclaration) {
+        for (const declarator of node.declarations) {
+            this.visit(declarator);
+        }
     }
 
+    // noinspection TsLint
+    protected visitVariableDeclarator(node: VariableDeclarator) {
+
```

Comment:
Add:
`if (node.init !== null) {
    this.visit(node.init);
}`
for consistency's sake